### ISSUE TITLE
.NET Aspire Plugin for Rider stopped working after upgrade to Rider 2024.01

### ISSUE THREAD

**Issue by tjementum** [JetBrains/aspire-plugin#108]
I upgraded my Rider to 2024.1, and now when I start my Aspire project using the plugin, 2 of my projects are not starting.

![CleanShot 2024-04-10 at 16 32 35](https://github.com/rafaelldi/aspire-plugin/assets/1560381/8091634d-d736-46f1-8f21-6965c03c285d)

If I start the AppHost without using the plugin, it seems to work fine. The logs are empty in both the Aspire Dashboard but I see this in AppHost console in Rider:

```
fail: Aspire.Hosting.Dcp.dcpctrl.ExecutableReconciler[0]
      run session could not be started  {"Executable": {"name":"account-management-api-2vkhlu8"}, "Reconciliation": 7, "Status": "500 Internal Server Error", "Body": ""}
fail: Aspire.Hosting.Dcp.dcpctrl.ExecutableReconciler[0]
      run session could not be started  {"Executable": {"name":"app-gateway-cuu95rg"}, "Reconciliation": 12, "Status": "500 Internal Server Error", "Body": ""}
````

This is an open-source project, so you can clone https://github.com/platformplatform/PlatformPlatform and run the `/application/PlatformPlatform.sln`to reproduce.

I happened to have a screenshot from yesterday, of how it looked before the update:

![CleanShot 2024-04-10 at 16 38 26](https://github.com/rafaelldi/aspire-plugin/assets/1560381/47289882-2fc1-42f2-8739-2c6de134ceb7)

**Comment by rafaelldi** [comment]
Hi! It looks similar to #102. A release with this fix (v0.4.3) is already published and will be available after Marketplace checks. Feel free to reopen if that doesn't help.

**Comment by tjementum** [comment]
It worked after upgrading to 0.5.0 and Rider 2024.1.
